overlay/src/components/organisms/widgets/TeamView.js

```diff
@@ -256,80 +256,67 @@ const TeamVideoAnimationWrapper = styled.div`
   align-items: center;
 `;
 
-const mediaTypeByUrl = (url) => {
-    if (url.startsWith("webrtc://")) {
-        return "webrtc";
-    }
-    if (url.endsWith(".svg") || url.endsWith(".jpg") || url.endsWith(".png") || url.endsWith(".gif")) {
-        return "img";
-    }
-    return "video";
-};
 
-const TeamVideo = ({ teamId, type, setIsLoaded }) => {
-    const dispatch = useDispatch();
-    const team = useSelector((state) => state.contestInfo.info?.teamsId[teamId]);
-    if (!team || !team.medias || !team.medias[type])
-        return null;
-    const mediaUrl = team.medias[type];
-    const mediaUrlType = mediaTypeByUrl(mediaUrl);
-    console.log(`TeamVideo ${mediaUrl} ${mediaUrlType}`);
-    return <TeamVideoAnimationWrapper>
-        {mediaUrlType === "img" && <TeamImageWrapper
-            src={mediaUrl} onLoad={() => setIsLoaded(true)}/>}
-        {mediaUrlType === "video" && <TeamVideoWrapper
-            src={mediaUrl}
-            onCanPlay={() => setIsLoaded(true)}
-            onError={() => setIsLoaded(false) || dispatch(pushLog("ERROR on loading image in Picture widget"))}
-            autoPlay
-            muted/>}
-        {mediaUrlType === "webrtc" && <TeamWebRTCVideoWrapper
-            url={mediaUrl.split("webrtc://", 2)[1]}
-            setIsLoaded={setIsLoaded}/>}
-    </TeamVideoAnimationWrapper>;
+const TeamViewPInPWrapper = styled.div`
+  position: absolute;   
+  width: ${({ sizeX }) => `${sizeX * 0.4}px`};
+  height: ${({ sizeX }) => `${sizeX * 0.5625 * 0.4}px`};
+  right: 0;
+  top: ${({ sizeX }) => `${sizeX * 0.5625 * 0.6}px`};
+`;
+
+const teamViewComponentRender = {
+    TaskStatus: ({ setIsLoaded, teamId }) => {
+        useEffect(() => {
+            setIsLoaded(true);
+        }, [teamId]);
+        return <ScoreboardColumn teamId={teamId}/>;
+    },
+    Photo: ({ setIsLoaded, url }) => {
+        return <TeamVideoAnimationWrapper>
+            <TeamImageWrapper src={url} onLoad={() => setIsLoaded(true)}/>
+        </TeamVideoAnimationWrapper>;
+    },
+    Video: ({ setIsLoaded, url }) => {
+        return <TeamVideoAnimationWrapper>
+            <TeamVideoWrapper
+                src={url}
+                onCanPlay={() => setIsLoaded(true)}
+                onError={() => setIsLoaded(false)}
+                autoPlay
+                muted/>
+        </TeamVideoAnimationWrapper>;
+    },
+    WebRTCConnection: ({ setIsLoaded, url }) => {
+        return <TeamVideoAnimationWrapper>
+            <TeamWebRTCVideoWrapper url={url} setIsLoaded={setIsLoaded}/>
```

Comment:
it's better if the prop to indicate that it's loaded is called onStatusChange or something like that. Callbacks should start with 'on"